core-services/image-mirroring/openshift/mapping_origin_4_11

```diff
@@ -261,6 +261,8 @@ registry.ci.openshift.org/origin/4.11:recycler quay.io/openshift/origin-recycler
 registry.ci.openshift.org/origin/4.11:redhat-subscription-injection-webhook quay.io/openshift/origin-redhat-subscription-injection-webhook:4.11 quay.io/openshift/origin-redhat-subscription-injection-webhook:4.11.0 quay.io/openshift/origin-redhat-subscription-injection-webhook:latest
 registry.ci.openshift.org/origin/4.11:sdn quay.io/openshift/origin-sdn:4.11 quay.io/openshift/origin-sdn:4.11.0 quay.io/openshift/origin-sdn:latest
 registry.ci.openshift.org/origin/4.11:secondary-scheduler-operator quay.io/openshift/origin-secondary-scheduler-operator:4.11 quay.io/openshift/origin-secondary-scheduler-operator:4.11.0 quay.io/openshift/origin-secondary-scheduler-operator:latest
+registry.ci.openshift.org/origin/4.11:self-node-remediation-index quay.io/openshift/origin-self-node-remediation-index:4.11 quay.io/openshift/origin-self-node-remediation-index:4.11.0 quay.io/openshift/origin-self-node-remediation-index:latest
+registry.ci.openshift.org/origin/4.11:self-node-remediation-operator quay.io/openshift/origin-self-node-remediation-operator:4.11 quay.io/openshift/origin-self-node-remediation-operator:4.11.0 quay.io/openshift/origin-self-node-remediation-operator:latest
 registry.ci.openshift.org/origin/4.11:service-ca-operator quay.io/openshift/origin-service-ca-operator:4.11 quay.io/openshift/origin-service-ca-operator:4.11.0 quay.io/openshift/origin-service-ca-operator:latest
```

Comment:
I see that only operator and index are here, don't we need bundle as well?

Reply:
no, the bundle is only used to get the operator's metadata into the index image. After that it's not needed anymore